morsedecoder.rb

```diff
@@ -0,0 +1,29 @@
+MORSE_CODE = {
+  '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E',
+  '..-.' => 'F', '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J',
+  '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N', '---' => 'O',
+  '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T',
+  '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y',
+  '--..' => 'Z'
+}.freeze
+
+def decode_char(morse_code)
+  MORSE_CODE[morse_code]
+end
+
+def decode_word(morse_code_word)
+  morse_characters = morse_code_word.split
+  decoded_characters = morse_characters.map do |morse_character|
+    decode_char(morse_character)
+  end
+  decoded_characters.join
+end
+
+def decode_sentence(morse_code_sentence)
```

Comment:
[OPTIONAL]
- It would look better if there was space between your words in a sentence. Currently, your answer looks like this.
![image](https://github.com/over-geek/decode_morse_code/assets/45460192/b8df145b-2602-4918-976a-d3fda6c28397)